configs/sim/gmoccapy/gmoccapy.ini

```diff
@@ -54,7 +54,15 @@ py = python3
 [RS274NGC]
 RS274NGC_STARTUP_CODE = G17 G21 G40 G43H0 G54 G64P0.005 G80 G90 G94 G97 M5 M9
 PARAMETER_FILE = sim.var
-SUBROUTINE_PATH = macros
+SUBROUTINE_PATH = ./macros
```

Comment:
Is this really needed?

Reply:
In some INI files is ../macros or ../../macros needed. I wanted to unify it.